plugins/generic/pln/templates/ping.tpl

```diff
@@ -9,12 +9,17 @@
 		<releaseDate>{$pluginVersion.date}</releaseDate>
 		<installed>{$pluginVersion.version->getDateInstalled()}</installed>
 		<current>{$pluginVersion.version->getCurrent()}</current>
+		<terms termsAccepted="{$termsAccepted|escape}">
```

Comment:
Isn't this `termsAccepted` attribute redundant? Is this just to summarize the status of the terms in aggregate?